webrtc/main.go

```diff
@@ -85,11 +87,15 @@ func main() {
 
 	inboundRTPPacket := make([]byte, 4096) // UDP MTU
 
+	var once sync.Once
+
 	// Read RTP packets forever and send them to the WebRTC Client
 	for {
 
 		n, _, err := listener.ReadFrom(inboundRTPPacket)
 
+		once.Do(func() { fmt.Print("houston we have a packet") })
+
 		if err != nil {
 			fmt.Printf("error during read: %s", err)
```

Comment:
Here too, although this one could be intentional

Reply:
yeah this one is intentional